hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -2957,7 +3074,9 @@ public String toString() {
                 + ", userContext=" + userContext
                 + ", memberAttributeConfig=" + memberAttributeConfig
                 + ", nativeMemoryConfig=" + nativeMemoryConfig
+                // todo: shall be removed in the future
```

Comment:
We cannot commit with such comment. Maybe mention that it is deprecated?

Reply:
It was just a trace to make it easier to find in the future, but thinking about it now, yeah doesn't make much sense. Removing these `todo`s